lib/ProductOpener/Display.pm

```diff
@@ -407,7 +407,7 @@ sub process_template($$$) {
 
 	# Display one taxonomy entry in the target language
 	$template_data_ref->{display_taxonomy_tag} = sub ($$) {
-		return display_taxonomy_tag($lc, $_[0], $_[1]);
+		return display_taxonomy_tag_name($lc, $_[0], $_[1]);
```

Comment:
This is ok I guess, I don't think we have templates where we do want the tag with the leading language code. We might in the future though, e.g. if we want to use panels to link to a list of products with a specific tag.

Another option is to change display_taxonomy_tag to display_taxonomy_tag_name in all templates.

Reply:
I will change to have a specific display_taxonomy_tag_name available in templates

Reply:
I've change that, to instead have two function and use appropriate one in knowledge panels.